### ISSUE TITLE
Pylint considers that there is no IsolatedAsyncioTestCase in unittest

### ISSUE THREAD

**Issue by socketpair** [PyCQA/pylint#4015]
```python
from asyncio import sleep
from unittest import IsolatedAsyncioTestCase

class Test(IsolatedAsyncioTestCase):
    async def test_xxx(self):
        await sleep(0.1)
        self.assertTrue(True)
```

shows:

```
tests/test_xxxx.py:4:0: E0611: No name 'IsolatedAsyncioTestCase' in module 'unittest' (no-name-in-module)
```

pylint from Fedora 33 (Python 3.9)

version 2.6.0-2.fc33

Actually  this module added in Python 3.8

**Comment by hippo91** [comment]
@socketpair  thanks for your report. I can reproduce it.

**Comment by gbrennon** [comment]
is there any fix for this false-positive?

**Comment by Pierre-Sassoulas** [comment]
No one worked on it yet, do you want to tackle it @gbrennon  ? :)

**Comment by jacobtylerwalls** [comment]
Same cause as #4060, fixed in https://github.com/PyCQA/astroid/pull/1127.

**Comment by DanielNoord** [comment]
Closing! 🛩️
